etc/vector/vector.d/90-output.toml.tmpl

```diff
@@ -98,6 +98,19 @@ source = '''
 '''
 {{end}}
 
+{{if conv.ToBool (getenv "DESKPRO_LOG_AGGREGATION" "false")}}
```

Comment:
If a new env is being added, should also add it to https://github.com/deskpro/docker-product-base/blob/main/usr/local/share/deskpro/container-var-reference.json for documentation purposes

Reply:
Or overload LOGS_EXPORT_TARGET? e.g. below "cloudwatch" is special, could add like "single_file" or something that means the same thing

Reply:
But also not sure this is what we want. e.g. a single file for access logs mixed in with PHP errors mixed in with deskpro application logs.

If you really wanted just a big stream of logs then you'd be better to just _not_ export logs, and use docker logs instead

Reply:
If I'm honest, I was mulling it over initially because in CI I was investigating something and was tracing through the multiple log files thinking it might make it a bit easier, to have a filtered central log. I'd added the condition in so it should just be apps that start with `deskpro*` not everything because I thought the same with access logs etc all mixed together but that may need a tweak. I was debating closing off the PR but it came up again in a conversation with CR too as something that may be handy.  
I've only rebased today as me or CR may test it at some point soon, if we get a moment, and tinker and/or check how much help it is before we go through finalising it i.e we might want to make the condition on the filter an env or if it's less help than we thought may just close off the PR etc, but it's raised as a draft PR currently in order to make contributions easier. 
I was kind of seeing this as an additional place to look when trying to hone in on a problem rather than a replacement for the other log files though

Reply:
I might expect on CI that disabling log export entirely might be preferable (i.e. use standard docker logging)